### DISCUSSION TITLE
I can't install tailwindcss when run "mix setup"

### DISCUSSION THREAD

**Discussion by marvel57** [zoonk/uneebee#176] [Bugs & Issues] [answered]
Please help me fix below error when I try to setup uneebee project with command "mix setup"

12:55:48.810 [debug] Downloading tailwind from https://github.com/tailwindlabs/tailwindcss/releases/download/v3.3.2/tailwindcss-linux-x64
** (RuntimeError) Couldn't fetch https://github.com/tailwindlabs/tailwindcss/releases/download/v3.3.2/tailwindcss-linux-x64: {:error, {:failed_connect, [{:to_address, {~c"github.com", 443}}, {:inet, [:inet], {:options, {:insufficient_crypto_support, {:"tlsv1.3", {:versions, [:"tlsv1.2", :"tlsv1.3"]}}}}}]}}

Here is my elixir (elixir -v)
Erlang/OTP 26 [erts-14.1.1] [source] [64-bit] [smp:2:2] [ds:2:2:10] [async-threads:1]

Elixir 1.17.0-dev (dde2a7c) (compiled with Erlang/OTP 26)

**Comment by ceolinwill** [comment]
Your logs seem to show you're using a dev version of Elixir (`1.17.0-dev`). Please, can you try using the latest stable version instead (`1.15.7`) to see if you still have this issue?

**Reply by marvel57** [reply, marked as answer]
Thank you for your response.
I have fixed it. I removed "tlsv1.3" crypto in file "deps/tailwind/lib/tailwind.ex" and this issue is fixed.
